webbpsf/utils.py

```diff
@@ -719,8 +719,9 @@ def to_griddedpsfmodel(HDUlist_or_filename=None, ext=0):
 
     # Convert header to meta dict
     header = header.copy(strip=True)
-    header.remove('COMMENT', remove_all=True)
-    header.remove('', remove_all=True)
+    header.pop('COMMENT', None)
+    header.pop('', None)
+    header.pop('HISTORY', None)
```

Comment:
Why is it desirable or necessary to delete these header keywords? I'm probably missing some simple explanation...

Reply:
The idea was that we don’t want these keys in a meta dictionary because they’re either unhelpful or incomplete. E.g. getting a meta entry of  `'': ('', '')` isn’t useful. And for the `comment`/`history` cards, right now you only get the first line rather than the entire entry and we determined that the `comment`/`history` cards don't contain important enough information to do the work to get the all the lines in the meta dict (the information in them can be found in documentation). But if you think that those lines are useful, I can make it so its all included.

Reply:
I don’t think they’re so useful as to be necessary. All the other keywords that do get preserved should maintain sufficient information about the parameters used in case someone later needs to investigate how a given PSF grid was generated, right?

Reply:
Yes exactly. There should be no information in the comment/history cards that can't be found elsewhere (either in other keys or in the webbpsf/photutils notebooks)